pkg/cmd/taskrun/delete_test.go

```diff
@@ -261,6 +261,64 @@ func TestTaskRunDelete(t *testing.T) {
 				},
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "tr0-8",
+				Namespace: "ns",
+				Labels:    map[string]string{"tekton.dev/clusterTask": "random"},
+			},
+			Spec: v1alpha1.TaskRunSpec{
+				TaskRef: &v1alpha1.TaskRef{
+					Name: "random",
+					Kind: v1alpha1.ClusterTaskKind,
+				},
+			},
```

Comment:
we can skip adding tests for v1alpha1 as the support is going to be removed sooner.
/cc @piyush-garg @pradeepitm12 @vdemeester

Reply:
:+1: